apps/vanilla/src/pages/blackjack/Classes/Player.ts

```diff
@@ -0,0 +1,28 @@
+import { WIN_SCORE } from '../constants';
+import { Subscriber } from '../Types/subscriber';
+import { Publisher } from '../Types/publisher';
+import { PlayerTurnResult } from '../Types/player-turn-result';
+
+import { Inspector } from './inspector';
+import { ResultData } from './result-displayer';
+
+/** Player . */
+export class Player extends Inspector implements Subscriber<PlayerTurnResult> {
+	/** Win status of each player .*/
+	public readonly winStatus: Publisher<boolean> = new Publisher<boolean>();
+
+	public constructor(private readonly playerIndex: number) {
+		super();
+	}
+
+	/** @param turnResult The result of current dice turn which has result and index of player . */
```

Comment:
All functions should have description. So functions without params can have singe line jscod, but function with params, should always have multiline jsdoc.
```suggestion
	/** 
	 * Update turn.
	 * @param turnResult The result of current dice turn which has result and index of player.
	 */
```